spring-cloud-data-core/src/main/java/org/springframework/cloud/data/core/dsl/StreamDslParser.java

```diff
@@ -255,32 +366,38 @@ private ChannelNode eatChannelReference(boolean tapAllowed) {
 		if (tapAllowed && firstToken.data.equalsIgnoreCase("tap")) {
 			if (tokens.peek(TokenKind.DOT)) {
 				if (channelScopeComponents.size() < 3) {
-					tokens.raiseException(firstToken.startPos, DSLMessage.TAP_NEEDS_THREE_COMPONENTS);
+					tokens.raiseException(firstToken.startPos,
+							DSLMessage.TAP_NEEDS_THREE_COMPONENTS);
 				}
 				String tokenData = channelScopeComponents.get(1).data;
 				// for Stream, tap:stream:XXX - the channel name is always indexed
 				// for Job, tap:job:XXX - the channel name can have "." in case of job notification channels
 				if (!tokenData.equalsIgnoreCase("stream") && !tokenData.equalsIgnoreCase("job")) {
```

Comment:
That is an ongoing discussion - we are most likely going to replace "job" with "task" globally - with jobs being a specific type of task that is based on Spring Batch, with its repositories, listeners, etc (but other simpler tasks may just be arbitrary code that is schedulable/launchable). That's beyond the scope of this PR, even though much of this code may be affected later. cc @mminella